src/index.js

```diff
@@ -1,45 +1,17 @@
 import './style.css';
 
-const toDos = [
-  {
-    description: 'wash the dishes',
-    completed: true,
-    index: 0,
-  },
-  {
-    description: 'complete To Do list project',
-    completed: true,
-    index: 1,
-  },
+import { getToDos, createToDo, addToDo } from './modules/addRemoveEdit.js';
 
-  {
-    description: 'Take a shower',
-    completed: false,
-    index: 2,
-  },
+const input = document.querySelector('.desc');
 
-  {
-    description: 'clean the car',
-    completed: true,
-    index: 3,
-  },
-
-  {
-    description: 'go for shopping',
-    completed: false,
-    index: 4,
-  },
-
-];
-const ul = document.querySelector('.list-items');
-toDos.forEach((todo) => {
-  const li = document.createElement('li');
-  const x = document.createElement('INPUT');
-  x.setAttribute('type', 'checkbox');
-  x.checked = todo.completed;
-  const y = document.createElement('span');
-  y.innerText = todo.description;
-  li.append(x, y);
-
-  ul.appendChild(li);
-});
\ No newline at end of file
+getToDos().forEach(createToDo);
+input.addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    const completed = false;
+    const description = document.querySelector('.desc').value;
+    const index = getToDos().length + 1;
+    const newTodo = createToDo({ description, completed, index });
```

Comment:
- [ ]  You should fix the issue, that when you check the item, it should `completed `should be turned to `true`
![image](https://user-images.githubusercontent.com/43877689/177845788-b1c5a99c-e3a1-4d29-8000-e39912975a78.png)